README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍
- [ ] It would be nice if you make your commit messages capitalized to make your pull request more professional and descriptive 😊